### DISCUSSION TITLE
B2B Next Version Backlogs

### DISCUSSION THREAD

**Discussion by tanya-ondc** [ONDC-Official/ONDC-RET-Specifications#127] [General]
Below are some of the features to be released in next release.

Current Version: v2.0.2
Next Version: v2.x

**Current Features:**

1. RFQ & PO
2. Prepaid payment collected by BAP/BPP
3. POD Payment
4. Offline Negotiation - Communication Channel
5. Price Slabs
6. Cancellation
7. Text based customisation
8. Exports - Incoterms.

_____
To be enabled in same version:

Seller Led Journey
Polygons

**Backlog Features:**

1. Negotiations and Auction
2. Slotted Delivery
3. Part Payments
4. Post-Fulfillment payments
5. Multi - Fulfillments
6. Order Level Discounts
7. Exports - B2B Prepaid payment
8. Part Cancellation
10. Multi-mode payments
11. Enabling Credit

B2B2C

**Comment by 8QuadTech** [comment]
Hi @tanya-ondc 
What is B2B2C here?
